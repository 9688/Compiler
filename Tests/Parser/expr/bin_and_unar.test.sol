
list
     "||"
          "&&"
               "++@"
                    a
               "=="
                    "&@"
                         x
                    2
          "=="
               "+"
                    "*@"
                         "*@"
                              x
                    4
               CALL
                    f
                    list
                         "-@"
                              4

