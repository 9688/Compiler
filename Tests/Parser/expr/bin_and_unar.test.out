юдд''
   юдд'||'
      цдд'&&'
      Ё  цдд'++@'
      Ё  Ё  юдд'a'
      Ё  юдд'=='
      Ё     цдд'&@'
      Ё     Ё  юдд'x'
      Ё     юдд'2'
      юдд'=='
         цдд'+'
         Ё  цдд'*@'
         Ё  Ё  юдд'*@'
         Ё  Ё     юдд'x'
         Ё  юдд'4'
         юдд'('
            цдд'f'
            юдд'-@'
               юдд'4'

